psalm.xml

```diff
@@ -7,15 +7,10 @@
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
     errorBaseline="baseline.xml"
 >
-    <plugins>
```

Comment:
Will removal of this not disable the extension?

Reply:
We still have plugins:
```
<plugins><pluginClass class="Psalm\SymfonyPsalmPlugin\Plugin"/><pluginClass class="Weirdan\DoctrinePsalmPlugin\Plugin"/></plugins></psalm>
```

Updated Psalm, required updating format of config file